frontend/src/sass/common/uclapi.scss

```diff
@@ -256,6 +256,17 @@ a:hover { color: $color-grey-light; }
   margin: 2%;
   min-height: fit-content;
   overflow: hidden;
+
+  img {
+    -webkit-filter: brightness(75%);
```

Comment:
@TuxedoFish should we add a generic filter: for non-webkit browsers, e.g. firefox?

Reply:
Yeh, there is no need for any of them to use webkit have taken that out!